e2e-tests/batch-mode-tests.sh

```diff
@@ -11,7 +11,7 @@ until [[ ${statuscode} -eq 200 ]]; do
 done
 echo "${print_prefix} OPENMRS SERVER STARTED SUCCESSFULLY"
 echo "${print_prefix} BUILD THE ANALYTICS PROJECT"
-mvn clean install
+mvn clean install -DskipTests=true 
 rm -rf /tmp/analytics-tests
 mktemp -d /tmp/analytics-tests
 echo "${print_prefix} RUN BATCH MODE WITH PARQUET SYNC"
```

Comment:
also, it is worth mentioning that we have 2  modes of the batch pipeline, and we might want to e2e both of them. However, I think we can address this in another ticket

Reply:
+1, let's create an issue for this and leave it for a future PR.